scripts/generate_embeddings.py

```diff
@@ -149,6 +178,16 @@ def got_whitespace(text: str) -> bool:
         else:
             print("skipping node without whitespace: " + node.__repr__())
 
+    runbook_documents = SimpleDirectoryReader(
+        args.runbooks,
+        recursive=True,
+        required_exts=[".md"],
```

Comment:
i guess this will cover avoiding embedding the images and LICENSE file.

probably still better if we don't even put the images into our repo since it just bloats our repo.

Reply:
done